src/test/resources/application-local.yaml

```diff
@@ -6,9 +6,12 @@ no.nav.security.jwt:
 
 ACCEPTED_CLIENTS: ${BA_SAK_CLIENT_ID}, ${EF_SAK_CLIENT_ID}
 
-spring.datasource.url: jdbc:postgresql://0.0.0.0:5432/familie-tilbake
-spring.datasource.username: postgres
-spring.datasource.password: test
+spring:
```

Comment:
det brukes for å kjøre familie-tilbake i lokal. Kan vi ikke hard-kode database url her?